plugin/core/protocol.py

```diff
@@ -46,6 +46,11 @@ class SignatureHelpTriggerKind:
     ContentChange = 3
 
 
+class InsertTextMode:
+    AsIs = 1
```

Comment:
Should I use lowercase property names to follow the spec more closely? 

```ts
export namespace InsertTextMode {
	export const asIs: 1 = 1;
        export const adjustIndentation: 2 = 2;
```

Or should I leave them capitalized. (to be consistent with other class types like the property keys in: DiagnosticSeverity, DiagnosticTag, CompletionItemTag...)